crossbar/webservice/static.py

```diff
@@ -155,8 +155,8 @@ def create(transport, path, config):
         #
         fallback = static_options.get('default_file')
         if fallback:
-            resource.childNotFound = ResourceFallback(path, config)
+            resource.childNotFound = ResourceFallback(os.path.join(static_dir, fallback))
```

Comment:
we can remove `ResourceFallback` altogether and just use `File` here directly

Reply:
done